### DISCUSSION TITLE
自定义title时，网页栏提示变成[object Object],如何去掉？

### DISCUSSION THREAD

**Discussion by Mr-Jies** [umijs/umi#12508] [Q&A]
### 🧐 问题描述 | Problem description

在`app.tsx` -- `RunTimeLayoutConfig` 中 配置的参数

```
    logo: <h1><img src={logo3} style={{ height: '32px' }}></img></h1>,
    title: <img src={logoTitle} style={{ height: '32px' }}></img>,
```

效果：

![image](https://github.com/ant-design/ant-design-pro/assets/56713068/79f66d47-3226-4844-9f1b-0f8072d5edd0)

如何把[object Object]这个东西给去掉？？？？

浏览器 | browser：chrome

**Comment by fz6m** [comment]
网页 title 必须是一个纯文本，不能是 react 节点，react 无法渲染到浏览器 native 上。
